src/entities.spec.js

```diff
@@ -0,0 +1,51 @@
+import main from './main'
+import * as usersData from './test/users'
+import * as userData from './test/user'
+
+main.initialize({
+  host: 'http://localhost:8080'
+})
+
+main.setCredentials({
+  username: global.username,
+  password: global.password,
+})
+
+const entities = main.entities()
+
+describe('toccojs', () => {
+  describe('entities', () => {
+    afterEach(() => {
+      fetch.resetMocks()
+    })
+
+    test('should list logged-in users', done => {
+      fetch.mockResponse(JSON.stringify(usersData))
+      entities.list('User').then(data => {
+        expect(data.length).toBe(2)
+        expect(fetch.mock.calls[0][1].headers['X-Business-Unit']).toBe('__n-u-l-l__')
```

Comment:
not need to test for the x-business-unit header in the tests for the `entities` file, as the business unit stuff is handled "globally" in the `main` file.

tests should be isolated and test only 1 thing (we don't care about the business unit in the entities.list() function).